### PR TITLE
Option to skip uploading a binary

### PR DESCRIPTION
**Pull request by JaviSoto**
This is useful if you use pilot for that, but want to use deliver to submit screenshots

### CODE REVIEW

**Comment by JaviSoto** [comment]
Don't merge yet, I'm going to test it first :)

File: lib/deliver/options.rb [hunk 1]
```diff
@@ -60,6 +60,11 @@ def self.available_options
                                      short_option: '-w',
                                      description: "Path to the folder containing the screenshots",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :skip_upload,
+                                     short_option: '-s',
+                                     description: "Skip uploading an ipa or pkg to iTunes Connect",
+                                     is_string: false,
+                                     optional: true),
```
Comment (KrauseFx) [inline lib/deliver/options.rb]: Instead of optional, set the default_value to false

File: lib/deliver/runner.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ def login
     def run
       verify_version if options[:app_version].to_s.length > 0
       upload_metadata
-      upload_binary if options[:ipa] || options[:pkg]
+      upload_binary if !options[:skip_upload] && (options[:ipa] || options[:pkg])
```
Comment (KrauseFx) [inline lib/deliver/runner.rb]: Let's move that to multi-line so it's clearer what it does
```ruby
has_binary = (options[:ipa] || options[:pkg])
if !options[:skip_upload] && has_binary
  upload_binary
end
````
:rocket:

Comment (JaviSoto) [inline lib/deliver/runner.rb]: Totally!

File: lib/deliver/options.rb (hunk 1 above)
Comment (JaviSoto) [inline lib/deliver/options.rb]: oh!

**Comment by JaviSoto** [comment]
Updated based on feedback

**Comment by KrauseFx** [comment]
Perfect, thanks @JaviSoto :rocket:

**Pull request merged**